### PR TITLE
Logging properly

### PR DESCRIPTION
**Pull request by joaogelado**
I made this pull request to help logging and bug reporting, so:
-Issues can be more easily resolved
-Debugging and testing can be more easy
-Errors are more understanding

I am using the native lib called ``logging``

### CODE REVIEW

File: TempYouTubeSpammerPurge.py [hunk 1]
```diff
@@ -0,0 +1,932 @@
+#!/usr/bin/env python3
```
Comment (ryanolee) [inline TempYouTubeSpammerPurge.py]: Should this file be here?

File: client_secret_98914997661-bbf8t1gdau8s4f5ek650o5cs7c6d3vi5.apps.googleusercontent.com (4).json:Zone.Identifier [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
```
Comment (ryanolee) [inline client_secret_98914997661-bbf8t1gdau8s4f5ek650o5cs7c6d3vi5.apps.googleusercontent.com (4).json:Zone.Identifier:1]: This does not seem like this should be committed?

**Comment by joaogelado** [comment]
I'm sorry, I forgot

**Comment by ThioJoe** [comment]
Ok quite a few additions so will take me some time to review

**Comment by joaogelado** [comment]
> Ok quite a few additions so will take me some time to review

No problem!

**Comment by joaogelado** [comment]
no one cares about this commit at this point

**Comment by ThioJoe** [comment]
Yea I had updated the main program so much that it would have been a ton of work to get the PR caught up. I might use some of the code if I add logging at some point, so if I do I'll add you as a co-author.

**Pull request closed without merging**